comfy_extras/nodes_string.py

```diff
@@ -0,0 +1,331 @@
+import re
+
+from comfy.comfy_types.node_typing import IO
+
+def normalize_path(path):
+    return path.replace('\\', '/')
+
+class StringConcatenate():
```

Comment:
I am not sure if the separator here is really necessary given we only support concat of 2 other strings. Can you find a reference of how existing node packs is handling the variable length input stacking and do that instead?

Ref: There is variable length input proposed in https://github.com/Comfy-Org/rfcs/pull/18, but currently it is not implemented yet.

Reply:
I removed separator for now, and we could add it back once we implement variable length input later